standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/MetaStoreDirectSql.java

```diff
@@ -2635,10 +2634,9 @@ private void dropPartitionsByPartitionIds(List<Long> partitionIdList) throws Met
         for (Object[] fields : sqlResult) {
           sdIdList.add(MetastoreDirectSqlUtils.extractSqlLong(fields[0]));
           Long colId = MetastoreDirectSqlUtils.extractSqlLong(fields[1]);
-          if (!columnDescriptorIdList.contains(colId)) {
-            columnDescriptorIdList.add(colId);
-          }
-          serdeIdList.add(MetastoreDirectSqlUtils.extractSqlLong(fields[2]));
+          columnDescriptorIdSet.add(colId);
+          Long serdeId = MetastoreDirectSqlUtils.extractSqlLong(fields[2]);
+          serdeIdSet.add(serdeId);
```

Comment:
Can you directly add the Long variable to the set instead of creating a 'serdeId' variable?

Reply:
Sure, will do.